lib/postprocess/processImage.js

```diff
@@ -130,6 +142,15 @@ module.exports = async function processImage(files, drawingEl, newRelIdCounterMa
     imageBuffer = Buffer.from(response.data)
   }
 
+  // check if the file is valid
```

Comment:
**note:** it's been a long time since this has been implemented so some uncertainty here. but essentially we did this to build resiliency into the rendering process. we frequently had users upload images that were corrupt, or we weren't able to compute thumbnails for a specific pdf file (because the pdf thumbnail could not be generated).

Reply:
Hmm. Got it. Corrupted images. In our standard template, we surround each image usage with `if` statement. If we do not surround it, and we do not provide the image, the export basically fails. Since there are many templates out there for each project, this seems like a must have.
I will try to replicate the corrupted images and test that scenario. Thanks!

Reply:
Exactly. Without this extension the export would simply fail.